spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/util/FormUtils.java

```diff
@@ -991,7 +1015,7 @@ private static void processDilutions(String dilutionMolarity, Library library, P
         ldi.setSecurityProfile(library.getSecurityProfile());
         ldi.setConcentration(Double.valueOf(dilutionMolarity));
         ldi.setCreationDate(new Date());
-        ldi.setDilutionCreator(u.getLoginName());
+        ldi.setCreator(u);
         if (!library.getLibraryDilutions().contains(ldi)) {
           library.addDilution(ldi);
           log.info("Added library dilution: " + ldi.toString());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Format specifiers should be used instead of string concatenation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3457)